eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version3/ChannelCodecs3.scala

```diff
@@ -257,6 +257,11 @@ private[channel] object ChannelCodecs3 {
             ("remotePerCommitmentSecrets" | byteAligned(ShaChain.shaChainCodec))
         })).as[Commitments]
 
+    val closingFeeratesCodec: Codec[ClosingFeerates] = (
+      ("preferred" | feeratePerKw) ::
+        ("min" | feeratePerKw) ::
+        ("max" | feeratePerKw)).as[ClosingFeerates]
+
```

Comment:
Exactly, this is used to store the fee range the node operator wants eclair to use (because `shutdown` may take a while if htlcs are pending, and we need their choice to survive a restart).